multivac/gather_data.py

```diff
@@ -161,6 +161,23 @@ def get_release_or_debug(log_file):
                 return 'True'
         return 'False'
 
+    @staticmethod
+    def detect_os_version(job_name):
```

Comment:
Should we detect macOS as well?

Reply:
Yes. With this PR we will detect it from job name - are there any jobs on osx with no `osx` in the job name?

Reply:
Ok, got it, thanks.